src/main/java/it/pagopa/selfcare/pagopa/backoffice/repository/WrapperRepository.java

```diff
@@ -1,7 +1,7 @@
 package it.pagopa.selfcare.pagopa.backoffice.repository;
 
 import it.pagopa.selfcare.pagopa.backoffice.entity.WrapperEntities;
-import it.pagopa.selfcare.pagopa.backoffice.entity.WrapperEntitiesOperations;
+import it.pagopa.selfcare.pagopa.backoffice.entity.WrapperEntities;
```

Comment:
This is a duplicated import, can you remove it?

Reply:
yes